docs/GettingStarted/HelmSetup.md

```diff
@@ -34,9 +34,11 @@ To install the chart with release name `devlake`,follow these steps:
 
     **Please make sure to keep the ENCRYPTION_SECRET safe as it is used to encrypt sensitive information in the database, such as personal access tokens and passwords. If ENCRYPTION_SECRET is lost, it may not be possible to decrypt this sensitive information.**
 
-3.  By default, the timezone is UTC. You can set your timezone via --set commonEnvs.TZ="your timezone",grafana.env.TZ="your timezone"
+3.  By default, the timezone is UTC. You can set your timezone via ```--set commonEnvs.TZ="your timezone",grafana.env.TZ="your timezone"```.
```

Comment:
@ZhangNing10 I presume what you want here is inline code, so maybe consider using ` instead of ```? Also, the style of the code block above doesn't look right, please see screenshot below:

<img width="2309" alt="Screen Shot 2023-11-08 at 7 56 30 AM" src="https://github.com/apache/incubator-devlake-website/assets/2908155/083bda70-f874-44ea-a356-117352827508">